Research Paper/BibTex Template/Semanic Security.tex

```diff
@@ -0,0 +1,100 @@
+ 
+\begin{figure}[htbp]
+\centerline{\includegraphics[width=0.5\textwidth]{./DFD.jpg}}
+\caption{Data Flow Diagram}
+\label{fig}
+\end{figure}
+
+This diagram shows the data flow of the system. As explained in the previous paragraphs, the information comes from Facebook and then is parsed through the tag and post information, which will later be used to extract the entities and processed to create the ontology with the different relationships those entities have. Finally, the ontology will be used to generate the nodes that will be used to populate the knowledge graph.
+
+\section{Results}
+The previously stated method resulted in the creation of a Knowledge graph displayed in the figure below. 
+
+\begin{figure}[H]
+\centerline{\includegraphics[width=0.5\textwidth]{./Knowledge_Graph_1.png}}
+\caption{Knowledge Graph}
+\label{fig1}
+\end{figure}
+
+In this Knowledge Graph, it can be seen four different types of nodes. The nodes in red show the different people the user interacts with, in this case, friends in the social media. The nodes in blue are the different entities extracted from the user posts. The node in lighter blue displays the user id from the social network. Finally, the nodes in yellow are the id of the interaction where the other nodes are extracted from. The ontologies can extract most of the useful information from the posts, however it can be seen that it does struggle with words that start in an upper-case letter. This is because it believes that capitalized words are important.
+	The following screenshot displays in more detail the knowledge graph. 
+
+\begin{figure}[H]
+\centerline{\includegraphics[width=0.5\textwidth]{./Knowledge_Graph_2.png}}
+\caption{Knowledge Graph}
+\label{fig2}
+\end{figure}
+
+The node expanded is the one with the circle around. In the right side of the screenshot, it is shown the post id, the type the post was detected as for the ontologies used, in this case the post is a “Post” type for the “soic” ontology and “Document” for the “ns1” document. It is also displayed comment from the post, and the date the post was created. 
+
+\section{Conclusion}
+
+
+\section{Further Works}
+Based on what it has been accomplished by creating knowledge graphs from information posted on social media accounts, further work could lead towards the notification to the users whos’ information posted online is related to their security questions. Once this is accomplished and proven to be successful, it could be expanded to other social media sites (Instagram, Twitter, Reddit…). 
```

Comment:
Based on what has been accomplished by creating knowledge graphs from information posted on social media accounts, further work could lead towards notifying users who have information posted online related to their security questions.

(e.g. Instagram, Twitter, Reddit)